mirrord/agent/src/entrypoint.rs

```diff
@@ -514,7 +514,7 @@ async fn start_agent(args: Args) -> Result<()> {
     } else {
         let cancellation_token = cancellation_token.clone();
         let is_mesh = args.is_mesh();
-
+        let (sniffer_init_tx, sniffer_init_rx) = tokio::sync::oneshot::channel::<bool>();
```

Comment:
```suggestion
        // We're using this to avoid crashing on old kernels when initializing the `RawSocketTcpCapture`.
        let (sniffer_init_tx, sniffer_init_rx) = tokio::sync::oneshot::channel::<bool>();
```